tinker/faculty_bios/__init__.py

```diff
@@ -106,6 +114,30 @@ def confirm(self):
     def faculty_bio_in_workflow(self):
         return render_template('faculty-bios/in-workflow.html')
 
+    @route('/toggle-courseleaf-user', methods=['post'])
+    def toggle_courseleaf(self):
+        data = self.base.dictionary_encoder.encode(json.loads(request.data))
+        faculty_bio_id = data['id']
+
+        page = self.base.read_page(faculty_bio_id)
+        asset, md, sd = page.get_asset()
+
+        # activate bio
+
+        added = find(sd, 'courseleaf-user', False)
+
+        if added == 'Yes':
+            update(sd, 'courseleaf-user', 'No')
+        else:
+            update(sd, 'courseleaf-user', 'Yes')
+        asset['page']['shouldBePublished'] = True
+        page.edit_asset(asset)
+        page.publish_asset()
```

Comment:
Similarly, we don't need to publish, since the content isn't changing.

Reply:
Removed